reason-react-native/src/apis/AccessibilityInfo.md

```diff
@@ -0,0 +1,72 @@
+---
+id: apis/AccesibilityInfo
+title: AccesibilityInfo
+wip: true
+---
+
+```reason
+type payload = {
```

Comment:
Shouldn't this be called `announcementFinishedPayload` (or maybe `announcementFinishedParams` to match other event types)?

Reply:
How about announcementResult or announcementOutcome?

Reply:
`announcementFinishedResult` seems a good compromise.
Not a fan of "params" that could be interpreted as an argument from the user.

Reply:
👍